### DISCUSSION TITLE
Cygwin

### DISCUSSION THREAD

**Discussion by knapsea** [nasa/fprime#861] [Q&A] [answered]
Hello,
Though we eventually plan to deploy on Linux, 
Development on Cygwin has many advantages as most of us have Windows Laptops . Windows Subsystem for Linux (WSL) is a option we considered but when compared to Cygwin is too bulky for now.

Is there a way to bring the Cygwin port back ? or any suggestion to develop when Linux is inaccessible.

Thanks
G

**Comment by LeStarch** [comment, marked as answer]
We use WSL or another Linux virtual machine.  Cigwin support was removed as most developers had easier access to WSL and maintenance across multiple versions of cigwin was becoming burdensome.
